common/changes/@itwin/frontend-tiles/eringram-graphicrepresentation-fix_2025-01-21-17-59.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/frontend-tiles",
+      "comment": "Update GraphicRepresentationFormat type and GraphicRepresentationStatus enum to reflect the latest Mesh Export Service docs.",
```

Comment:
should probably mention/update that GraphicRepresentationStatus is no longer an enum but a string literal type

Reply:
Good point, added